README.md

```diff
@@ -1,16 +1,30 @@
 # Package Support Framework
-This project provides tools, libraries, documentation and samples for creating app-compat fixups to enable classic Win32 applications to be distributed and executed as packaged apps.
+This project provides tools, libraries, documentation and samples for creating app-compat fixups to enable classic Win32 applications to be distributed and executed as MSIX packaged apps.
+
+Here are some common examples where you can find the PSF useful:<br/>
+	1. Your app can't find some .dlls when launched, you may need to set your current working directory.

```

Comment:
Remove the tabs from 1., 2., and 3. and add a line break before 1. to make this look like a list.  You can use https://stackedit.io/app# to see what the readme will look like.